packages/main/src/DateTimePickerPopover.hbs

```diff
@@ -23,7 +23,7 @@
 			timestamp="{{_calendarTimestamp}}"
 			.selectionMode="{{_calendarSelectionMode}}"
 			.minDate="{{minDate}}"
-			.maxDate="{{maxDate}}"
+			.maxDate="{{maxDate}}"			
```

Comment:
revert the extra tabs here as they will create confusing git history for this line